src/main/java/seedu/address/ui/BrowserPanel.java

```diff
@@ -67,7 +67,7 @@ public void freeResources() {
 
     @Subscribe
     private void handlePersonPanelSelectionChangedEvent(PersonPanelSelectionChangedEvent event) {
-        logger.info(LogsCenter.getEventHandlingLogMessage(event));
+        //logger.info(LogsCenter.getEventHandlingLogMessage(event));
```

Comment:
Remove commented code before merging any code to your main branch(es). If the purpose/goal of commenting code out is to "preserve the code" if something goes wrong, then that's the role of git.